src/Platform/Microsoft.Testing.Platform/Requests/ConsoleTestExecutionFilterFactory.cs

```diff
@@ -1,28 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using Microsoft.Testing.Platform.CommandLine;
-using Microsoft.Testing.Platform.Helpers;
-using Microsoft.Testing.Platform.Resources;
-
-namespace Microsoft.Testing.Platform.Requests;
-
-internal sealed class ConsoleTestExecutionFilterFactory(ICommandLineOptions commandLineService) : ITestExecutionFilterFactory
-{
-    private readonly ICommandLineOptions _commandLineService = commandLineService;
-
-    public string Uid => nameof(ConsoleTestExecutionFilterFactory);
-
-    public string Version => AppVersion.DefaultSemVer;
-
-    public string DisplayName => PlatformResources.ConsoleTestExecutionFilterFactoryDisplayName;
-
-    public string Description => PlatformResources.ConsoleTestExecutionFilterFactoryDescription;
```

Comment:
This resource is no longer used. Could you delete it as well?

Reply:
Deleted